bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ProcessValidator.java

```diff
@@ -33,8 +36,38 @@ public void validate(Process element, ValidationResultCollector validationResult
 
     final Collection<StartEvent> topLevelStartEvents =
         element.getChildElementsByType(StartEvent.class);
-    if (topLevelStartEvents.size() != 1) {
-      validationResultCollector.addError(0, "Must have exactly one start event");
+    if (topLevelStartEvents.size() == 0) {
+      validationResultCollector.addError(0, "Must have atleast one start event");
+    } else {
+      final Collection<StartEvent> noneStartEvents =
```

Comment:
This collection is called `noneStartEvents` but it collects both timer and none start events, as far as I can tell. So I would rename it to reflect what it is, i.e. `uniqueStartEvents` or some such, meaning they can't have multiple.